src/state/actions/register.ts

```diff
@@ -78,9 +78,6 @@ export const registerClient = async (store, client: ClientRegisterProps) => {
 			[userIdentity]: {
 				email,
 				password,
-				first_name: firstName,
-				last_name: lastName,
-				account_status: 'pending',
```

Comment:
These fields don't exist in the db for the client so I removed them.  If there are other fields required for clients, we'll need to add them here.